### DISCUSSION TITLE
About the discharge mode on and off

### DISCUSSION THREAD

**Discussion by AKmoex** [AppHouseKitchen/AlDente-Battery_Care_and_Monitoring#856] [Q&A] [answered]
Hi, I am trying AlDente free, I don't know how to distinguish the discharge mode on and off, for example the two pictures below, which one is the discharge mode on and which one is the discharge mode off?

<img width="405" alt="截屏2022-12-17 11 55 58" src="https://user-images.githubusercontent.com/63089888/208223384-748c1d1c-bb7e-4048-8278-ead7403f49c5.png">
<img width="398" alt="截屏2022-12-17 11 56 07" src="https://user-images.githubusercontent.com/63089888/208223393-402f2979-e72c-480b-abfa-9103028ee769.png">

**Comment by MatthiasKerbl** [comment, marked as answer]
Hi @AKmoex,

In the first picture, Discharge is off and in the second picture, Discharge is on.

Cheers,
Matthias

**Reply by AKmoex** [reply]
Got it, thank you
